cfgov/unprocessed/js/molecules/Autocomplete.js

```diff
@@ -121,23 +135,58 @@ function Autocomplete( element, opts ) {
       inputCoords.left ) + 'px';
   }
 
+  /**
+   * Toggles the error state and message that's shown when the
+   * max search term length is hit
+   */
+  function _toggleMaxLengthError() {
+    _maxLengthExceeded = _searchTerm.length >= _settings.maxChars ? true :
+      false;
+    if ( _maxLengthExceeded ) {
+      _input.classList.add( ERROR_CLASS );
+
+      /* If you type fast enough, search results from a letter or two back can
+         cover up the max length error, so we'll wait a bit before hiding */
+      setTimeout( _hide, _settings.delay );
+    } else {
+      _input.classList.remove( ERROR_CLASS );
+    }
+    _this.dispatchEvent( 'maxCharacterChange', {
+      maxLengthExceeded: _maxLengthExceeded
+    } );
+  }
+
+  /**
+   * Event handler for input into the _input element
+   */
+  function _handleInput() {
+    _searchTerm = this.value;
```

Comment:
I'd set this through the event parameter. 
Without testing I think it would be:
```
function _handleInput( evt ) {
  _searchTerm = evt.target.value;
```